src/platformdirs/windows.py

```diff
@@ -101,7 +108,11 @@ def get_win_folder_from_env_vars(csidl_name: str) -> str:
         raise ValueError(f"Unknown CSIDL name: {csidl_name}")
     result = os.environ.get(env_var_name)
```

Comment:
Without CSIDL_PERSONAL in the above map wouldn't this error be raised and never get to the fallback? I feel we should special case CSIDL_PERSONAL before that map, something like:

```python
if csidl_name == "CSIDL_PERSONAL": #  does not have an environment name
   return os.path.join(os.path.normpath(os.environ["USERPROFILE"]), "Documents")

```